src/main/java/cubicchunks/worldgen/generator/BasicCubeGenerator.java

```diff
@@ -52,11 +52,6 @@ public void generateColumn(Column column) {
 				Coords.cubeToMinBlock(column.getX()),
 				Coords.cubeToMinBlock(column.getZ()),
 				Cube.SIZE, Cube.SIZE);
-
-		byte[] abyte = column.getBiomeArray();
-		for (int i = 0; i < abyte.length; ++i) {
-			abyte[i] = (byte) Biome.getIdForBiome(this.columnBiomes[i]);
```

Comment:
`Chunk.getBiomeArray` returns the *internal* biome array and if you modify that array - you modify biomes in that chunk.